test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -628,32 +682,25 @@ static int check_alg_null(void)
 	return check_alg_support(ODP_CIPHER_ALG_NULL, ODP_AUTH_ALG_NULL);
 }
 
+#define ARRAY_SIZE(a) (sizeof(a) / sizeof(a[0]))
```

Comment:
Checkpatch takes issue with this:
```
WARNING: Prefer ARRAY_SIZE(a)
#303: FILE: test/validation/api/crypto/odp_crypto_test_inp.c:671:
+#define ARRAY_SIZE(a) (sizeof(a) / sizeof(a[0]))
total: 0 errors, 1 warnings, 0 checks, 893 lines checked
NOTE: Ignored message types: AVOID_EXTERNS BIT_MACRO COMPARISON_TO_NULL DEPRECATED_VARIABLE NEW_TYPEDEFS PREFER_PRINTF PREFER_SCANF SPLIT_STRING SSCANF_TO_KSTRTO VOLATILE
0001-validation-crypto-print-untested-capabilities.patch has style problems, please review.
```
This seems to be another instance of a Linux kernel macro? False positive.